sci-mathematics/rstudio/files/rstudio-0.99.486-systemd.patch

```diff
@@ -0,0 +1,38 @@
+diff -Naur /tmp/rstudio.org/work/rstudio-0.99.486/src/cpp/server/CMakeLists.txt ./work/rstudio-0.99.486/src/cpp/server/CMakeLists.txt
+--- /tmp/rstudio.org/work/rstudio-0.99.486/src/cpp/server/CMakeLists.txt	2015-12-10 19:29:24.408267862 +0100
++++ ./work/rstudio-0.99.486/src/cpp/server/CMakeLists.txt	2015-12-10 19:35:24.660194216 +0100
+@@ -208,17 +208,12 @@
+            DESTINATION ${DISTRO_SHARE}/${RSERVER_UPSTART_DIR})
+            
+     # install configured systemd profile
+-   set(RSERVER_SYSTEMD_DIR "extras/systemd")
++   set(RSERVER_SYSTEMD_DIR "lib/systemd/system")
+    set(RSERVER_SYSTEMD_PROFILE "${RSERVER_SYSTEMD_DIR}/rstudio-server.service")
+    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/${RSERVER_SYSTEMD_PROFILE}.in
+                   ${CMAKE_CURRENT_BINARY_DIR}/${RSERVER_SYSTEMD_PROFILE})
+    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${RSERVER_SYSTEMD_PROFILE}
+            DESTINATION ${RSERVER_SYSTEMD_DIR})
+-   set(RSERVER_SYSTEMD_PROFILE_REDHAT "${RSERVER_SYSTEMD_DIR}/rstudio-server.redhat.service")
+-   configure_file(${CMAKE_CURRENT_SOURCE_DIR}/${RSERVER_SYSTEMD_PROFILE_REDHAT}.in
+-                  ${CMAKE_CURRENT_BINARY_DIR}/${RSERVER_SYSTEMD_PROFILE_REDHAT})
+-   install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${RSERVER_SYSTEMD_PROFILE_REDHAT}
+-           DESTINATION ${RSERVER_SYSTEMD_DIR})
+ 
+ endif()
+ 
+diff -Naur /tmp/rstudio.org/work/rstudio-0.99.486/src/cpp/server/lib/systemd/system/rstudio-server.service.in ./work/rstudio-0.99.486/src/cpp/server/lib/systemd/system/rstudio-server.service.in
+--- /tmp/rstudio.org/work/rstudio-0.99.486/src/cpp/server/lib/systemd/system/rstudio-server.service.in	1970-01-01 01:00:00.000000000 +0100
++++ ./work/rstudio-0.99.486/src/cpp/server/lib/systemd/system/rstudio-server.service.in	2015-10-07 16:17:51.000000000 +0200
+@@ -0,0 +1,12 @@
++[Unit]
++Description=RStudio Server
++
++[Service]
++Type=forking
++ExecStart=${CMAKE_INSTALL_PREFIX}/bin/rserver
++ExecStop=/usr/bin/killall -TERM rserver
```

Comment:
And no. Why do you need that anyway? systemd should determine leader correctly anyway and kill it rather than random processes throughout the system.